setup.ts

```diff
@@ -570,6 +570,47 @@ export async function twoFactorAuth(): Promise<void> {
     updateEnvVariable(config);
   }
 }
+//Import sample data
+/**
+ * The function `importDefaultOrganization` will import the default organization
```

Comment:
1. Move the data importation logic to a file under `src/setup` for better readability
2. Setup.ts is too long at over 900 lines

Reply:
Remember this too